tests/unit/test_session_legacy.py

```diff
@@ -708,13 +708,19 @@ def test_create_client_verify_param_overrides_all(self, client_creator):
             self.assertEqual(call_kwargs['verify'], 'verify-certs.pem')
 
     @mock.patch('botocore.client.ClientCreator')
-    def test_create_client_use_no_api_version_by_default(self, client_creator):
+    def test_create_client_use_no_api_version_by_default(
+        self,
+        client_creator,
+    ):
         self.session.create_client('myservice', 'us-west-2')
         call_kwargs = client_creator.return_value.create_client.call_args[1]
         self.assertEqual(call_kwargs['api_version'], None)
 
     @mock.patch('botocore.client.ClientCreator')
-    def test_create_client_uses_api_version_from_config(self, client_creator):
+    def test_create_client_uses_api_version_from_config(
+        self,
+        client_creator,
+    ):
```

Comment:
Why are all of these changes here? Can you remove? I'm guessing your local formatter is a bit different than our.

Reply:
Hmm. I'm only using the pre-commit hook, but I think I touched these in a previous commit that was rebased and reverted. Will fix it though!